src/langsmith/trace-with-autogen.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Trace with AutoGen
+sidebarTitle: AutoGen
+---
+
+LangSmith can capture traces generated by AutoGen using OpenInference's AutoGen instrumentation. This guide shows you how to automatically capture traces from your AutoGen multi-agent conversations and send them to LangSmith for monitoring and analysis.
+
+## Installation
+
+Install the required packages using your preferred package manager:
+
+<CodeGroup>
+
+```bash pip
+pip install langsmith autogen openinference-instrumentation-autogen openinference-instrumentation-openai
+```
+
+```bash uv
+uv add langsmith autogen openinference-instrumentation-autogen openinference-instrumentation-openai
+```
+
+</CodeGroup>
+
+<Info>
+Requires LangSmith Python SDK version `langsmith>=0.4.26` for optimal OpenTelemetry support.
+</Info>
+
+## Setup
+
+### 1. Configure environment variables
+
+Set your API keys and project name:
+
+<CodeGroup>
+
+```bash Shell
+export LANGSMITH_API_KEY=<your_langsmith_api_key>
+export LANGSMITH_PROJECT=<your_project_name>
+export OPENAI_API_KEY=<your_openai_api_key>
+```
+
+</CodeGroup>
+
+### 2. Configure OpenTelemetry integration
+
+In your AutoGen application, import and configure the LangSmith OpenTelemetry integration along with the AutoGen and OpenAI instrumentors:
+
+```python
+from langsmith.integrations.otel import configure
+from openinference.instrumentation.autogen import AutogenInstrumentor
+from openinference.instrumentation.openai import OpenAIInstrumentor
+
+# Configure LangSmith tracing
+configure(project_name="autogen-demo")
+
+# Instrument AutoGen and OpenAI calls
+AutogenInstrumentor().instrument()
+OpenAIInstrumentor().instrument()
+```
+
+<Note>
+You do not need to set any OpenTelemetry environment variables or configure exporters manually—`configure()` handles everything automatically.
+</Note>
+
+### 3. Create and run your AutoGen application
+
+Once configured, your AutoGen application will automatically send traces to LangSmith:
```

Comment:
Suggestion: Add a brief sentence mentioning the additional code here, just really high-level "this includes the example application that..."
I think that helps to avoid the code dump feeling.
This suggestion goes for each of the pages on this step.